frontend-react/src/pages/submissions/SubmissionDetails.test.tsx

```diff
@@ -23,26 +28,48 @@ const timeRegex: RegExp = /\d{1,2}:\d{2}/;
 const mockData: ActionDetailsResource = new TestResponse(
     ResponseType.ACTION_DETAIL
 ).data;
-jest.mock("rest-hooks", () => ({
-    useResource: () => {
-        return mockData;
-    },
-    /* Must return children when mocking, otherwise nothing inside renders */
-    NetworkErrorBoundary: ({ children }: { children: JSX.Element[] }) => {
-        return <>{children}</>;
-    },
-}));
-
-describe("SubmissionDetails", () => {
+
+const renderWithResolver = (ui: ReactElement, fixtures: Fixture[]) =>
+    renderWithRouter(
+        <CacheProvider>
+            <NetworkErrorBoundary>
+                <Suspense fallback="suspended in test render">
+                    <MockResolver fixtures={fixtures}>{ui}</MockResolver>
+                </Suspense>
+            </NetworkErrorBoundary>
+        </CacheProvider>
+    );
```

Comment:
This will need to be de-duped with the version of the same helper [here](https://github.com/CDCgov/prime-reportstream/blob/76213ae9f3c641ddcd6aa40fefbb3fdb8342abf0/frontend-react/src/pages/submissions/SubmissionTable.test.tsx#L23). I think this one is better because it wraps the component under test with `NetworkErrorBoundary` and `Suspense`, which means we can test components that don't have those wrappers without getting warnings or errors.